### DISCUSSION TITLE
Multi Transactions Update

### DISCUSSION THREAD

**Discussion by DevAnyKR** [moneymanagerex/moneymanagerex#5941] [Q&A] [answered]
I need to change the income type to transfer type in Multi Transactions Update, but only To account can be specified. In the end, it is treated as an expense.
I want to specify both From and To. This is because it becomes a transfer (income).

This is because importing from CSV does not handle transfer types. So I have to do multiple edits.

![스크린샷 2023-05-25 163449](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/c25b217a-3c03-4bc2-813f-bb222a9cf1e9)

is bug? is missing a feature?

**Comment by tactilis** [comment]
>  I want to specify both From and To. This is because it becomes a transfer (income).

The From in the transfer will be the Account in which the transaction resides. Why do you need to change it?

**Comment by tactilis** [comment, marked as answer]
@DevelopmentAnything 

You'll find the rules about multi-transaction edit and transfers explained in this issue: https://github.com/moneymanagerex/moneymanagerex/issues/3897

You'll need to read the whole thread because the implementation was refined as a result of discussion in the issue.

**Reply by DevAnyKR** [reply]
https://github.com/moneymanagerex/moneymanagerex/issues/3897, This is an expected controversy.
The trick is to move the transaction to the expense account and then change it to the transfer type.

Editing one by one works fine. But multi-edit...😱

**Reply by tactilis** [reply]
You said:

> This is because importing from CSV does not handle transfer types.

That is correct. But all you need to do in to make sure you import the CSV transactions into the correct account in the first place. Then multi-edit for Transfers is easy.

There's no need for:
> The trick is to move the transaction to the expense account

**Reply by DevAnyKR** [reply]
@tactilis Then I might end up using Access and Excel again instead of MMEX. 🤣😁😱
